apps/scorecard/server/main.js

```diff
@@ -44,13 +44,20 @@ const doPlotParams = function () {
                 'these-days-of-the-week': ['Once'],
                 'these-days-of-the-month': ['Once'],
                 'these-months': ['Once'],
-                'dates': ['Recurring']
+                'scorecard-ends-on': ['Once'],
+                'dates': ['Recurring'],
 
             },
             controlButtonVisibility: 'block',
             displayOrder: 1,
             displayPriority: 1,
-            displayGroup: 1
+            displayGroup: 1,
+            tooltip: `The schedule mode, 'Once' means that this scorecard will be
```

Comment:
I think in the future we should explore ways for these messages to be in common code, so we don't have to copy and paste them into each app.

Reply:
Good catch, We should do that the way internationalization is often done, i.e. put a ne type in matstypes.tooltips.whatnot that contains each seperate and common message.

Reply:
I can do this today and push that change to this branch as well since we are waiting for the merge. This is a good idea.